index.js

```diff
@@ -1,110 +1,139 @@
-'use strict'
+'use strict';
 
 const koa = require('koa');
 const koaBody = require('koa-body');
 const GithubWebhookHandler = require('koa-github-webhook-handler');
 const github = require('./github');
 const bumpVersion = require('semver-increment');
+const Promise = require('bluebird');
 
-let app = koa();
+const app = koa();
 
-let githubWebhookHandler = new GithubWebhookHandler({
+const githubWebhookHandler = new GithubWebhookHandler({
   path: '/webhook',
   secret: process.env.SEANBOT_WEBHOOK_SECRET,
 });
 
-githubWebhookHandler.on('issue_comment', (event) => {
-  const payload = event.payload;
-
-  //judge if the comment is from pull requests
-  const pullRequest = payload['issue']['pull_request'];
-  if (!pullRequest) {
-    return;
+function getMasksFromCommentBody(commentBody) {
+  let MAJOR = 0;
+  let MINOR = 0;
+  let PATCH = 0;
+  if (commentBody.match('major')) {
+    MAJOR = 1;
+  } else if (commentBody.match('minor')) {
+    MINOR = 1;
+  } else if (commentBody.match('patch')) {
+    PATCH = 1;
+  } else {
+    return null;
   }
-
-  const commentBody = payload['comment']['body'].toLowerCase();
-
-  if (commentBody.match('@seanbot') && commentBody.match('bump')) {
-
-    function getMasksFromCommentBody() {
-      let MAJOR = 0;
-      let MINOR = 0;
-      let PATCH = 0;
-      if (commentBody.match('major')) {
-        MAJOR = 1;
-      } else if (commentBody.match('minor')) {
-        MINOR = 1;
-      } else if (commentBody.match('patch')) {
-        PATCH = 1;
-      } else {
-        return;
-      }
-      const masks = [MAJOR, MINOR, PATCH];
-      return masks;
-    }
-
-    function getMsgFormPrUrl() {
-      const list = pullRequest['url'].split('/');
-      return {
-        user: list[4],
-        repo: list[5],
-        number: list[7]
-      };
-    }
-
-    const masks = getMasksFromCommentBody();
-    if (!masks) {
-      return;
-    }
-
-    github.pullRequests.get(getMsgFormPrUrl(), (err, res) => {
+  const masks = [MAJOR, MINOR, PATCH];
+  return masks;
+}
+
+function getMsgFormPrUrl(url) {
+  const list = url.split('/');
+  return {
+    user: list[4],
+    repo: list[5],
+    number: list[7],
+  };
+}
+
+function getMsgFormHtmlUrl(url) {
+  const list = url.split('/');
+  return {
+    user: list[3],
+    repo: list[4],
+    branch: list[6],
+  };
+}
+
+function getPullRequests(url) {
+  return new Promise((resolve, reject) => {
```

Comment:
use [es6-promisify](https://github.com/digitaldesignlabs/es6-promisify) to simplify this.